tests/integration/integration_config.yml

```diff
@@ -0,0 +1,8 @@
+vcenter_hostname: "testvcenter"
```

Comment:
where do you set those attributes? for example if we need DC ?

Reply:
Currently it must be added manually here.